### PR TITLE
Support incrementallyLoadTextures property for 3D model in CZML

### PR DESCRIPTION
**Pull request by ezze**
In [Cesium 1.16](http://cesiumjs.org/2015/12/01/Cesium-version-1.16-released/)

> Added support for incrementally loading a glTF model’s textures.

that is default behavior. In order to show models only when textures are loaded (as it was in previous releases) one should set `incrementallyLoadTextures` property of [Model](http://cesiumjs.org/Cesium/Build/Documentation/Model.html) to `false` like it's proposed [here](https://groups.google.com/d/msg/cesium-dev/WT-0NM4bxC4/O74C6JUVBwAJ):

    var model = Model.fromGltf({
        url: 'model.gltf',
        incrementallyLoadTextures: false
    });

Unfortunately, it doesn't work if a model is specified as CZML packet:

    {
        "position": {
            "cartographicDegrees": [2.82519, 41.9874, -50]
        },
        "model": {
            "gltf": "model.gltf",
            "incrementallyLoadTextures": false
        }
    }

This pull request adds `incrementallyLoadTextures` property to [ModelGraphics](http://cesiumjs.org/Cesium/Build/Documentation/ModelGraphics.html) and supports it in [ModelVisualizer](http://cesiumjs.org/Cesium/Build/Documentation/ModelVisualizer.html) and [CzmlDataSource](http://cesiumjs.org/Cesium/Build/Documentation/CzmlDataSource.html).

### CODE REVIEW

**Comment by pjcozzi** [comment]
Good eye, thanks @ezze!

In CONTRIBUTORS.md, are you submitting this as an individual?  We have a corporate CLA from you.  Is that the CLA you mean to contribute this under?

**Comment by ezze** [comment]
@pjcozzi, I want to contribute as individual developer now, so, if you need individual CLA from me, I'm ready to sign it.

**Comment by pjcozzi** [comment]
Ah, gotcha.  Yes, we need an individual CLA.  Here is the info: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla

**Comment by ezze** [comment]
@pjcozzi, sent to cla@agi.com.

**Comment by pjcozzi** [comment]
Thanks, we got it!

This is ready to be reviewed.

File: Source/DataSources/ModelGraphics.js [hunk 1]
```diff
@@ -104,6 +107,15 @@ define([
         maximumScale : createPropertyDescriptor('maximumScale'),
 
         /**
+         * Get or sets the boolean Property specifying whether textures
+         * may continue to stream in after the model is loaded.
+         * {@link ModelGraphics#incrementallyLoadTextures}.
```
Comment (mramato) [inline Source/DataSources/ModelGraphics.js]: This link line is unnecessary.  I assume it's copied from `maximumScale` above, but that's linking to `minimumPixelSize`, this is linking to itself.

**Comment by mramato** [comment]
Just that one comment and I'll merge this.  Thanks for the PR @ezze!

**Comment by ezze** [comment]
@mramato, wrong link is removed.

**Comment by mramato** [comment]
Awesome. Thanks for the contribution!

**Pull request merged**